tests/factory_submit_request_tests.py

```diff
@@ -61,7 +61,7 @@ def setUp(self):
         FreezeCommand(self.wf.api).perform(STAGING_PROJECT_NAME)
 
         # Create the submit request
-        self.request = self.wf.create_submit_request(DEVEL_PROJECT, 'wine')
+        self.request = self.wf.create_submit_request(DEVEL_PROJECT, 'wine', add_commit=False)
```

Comment:
funny part is that I use `add_commit=True` as default to be backwards compatible